app/controllers/initiatives_controller.rb

```diff
@@ -1,13 +1,21 @@
 # frozen_string_literal: true
 
 class InitiativesController < ApplicationController
-  before_action :find_initiative, only: [:edit, :update, :destroy]
+  before_action :find_initiative, only: [:edit, :show, :update, :destroy]
   before_action :authenticate_user!
 
-  def index
+  def index  
     populate_initiative_sets
```

Comment:
Could `populate_initiative_sets` be pulled up to a `before_action` for the places its used?

Reply:
move up to before_action